src/Test/Legacy/WebServerListener.php

```diff
@@ -0,0 +1,144 @@
+<?php
+
+/*
+ * This file is part of the FOSHttpCache package.
+ *
+ * (c) FriendsOfSymfony <http://friendsofsymfony.github.com/>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace FOS\HttpCache\Test\Legacy;
+
+use FOS\HttpCache\Test\WebServerListenerTrait;
+use PHPUnit\Framework\TestListener;
+
+/**
+ * A PHPUnit test listener that starts and stops the PHP built-in web server.
+ * This legacy version is for PHPUnit 5.x (min 5.4.4 required, due to FC layer) 
```

Comment:
styleci complains about missing `.` at the end. please also add a blank line before this line.